src/js/utils/styles.js

```diff
@@ -459,3 +459,14 @@ export const plainInputStyle = css`
   outline: none;
   border: none;
 `;
+
+export const getPlainStyle = plain => {
+  if (plain === 'full') {
```

Comment:
Along the same lines, this behavior is very specific to TextInput hence the logic should be placed TextInput.

Reply:
I moved getPlainStyle to just be in TextInput